apps/client/src/router.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactElement } from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+
+import { NotFound } from './pages/NotFound';
+import { GuardedRoute } from './utils/guardedRoute';
+import { Home, Layout, Leaderboard, Race, Room } from './pages';
+
+export function Router(): ReactElement {
```

Comment:
Shall we go for a name like `AppRouter` ?

Reply:
renamed this in #114 ab14c6e1287fc13ba3a92c9facc56607020b84ff